dump-scripts/python/setup_eon2_json.py

```diff
@@ -0,0 +1,88 @@
+import collections
+import json
+import os
+import sys
+
+"""
+This script transforms the account data dumped from Eon in the format requested for the migration
+to Horizen 2.0.
+It takes as input:
+ - the json file with the data dumped from Eon
+ - the json file with the list of Eon delegators and their stakes
+ - the output filename to be generated
+
+It creates a json file with the data from Eon in alphabetical order.
+Only accounts that aren't smart contracts are saved in the file.
+"""
+
+
+if len(sys.argv) != 4:
+	print(
+		"Usage: python3 {} <Eon dump file name> <Eon stakes file name> <output_file>"
+		.format(os.path.basename(__file__)))
+	sys.exit(1)
+
+eon_dump_file_name = sys.argv[1]
+eon_stakes_file_name = sys.argv[2]
+result_file_name = sys.argv[3]
+
+with open(eon_dump_file_name, 'r') as eon_dump_file:
+	eon_dump_data = json.load(eon_dump_file)
+
+results = {}
+smart_contract_list = []
+
+total_balance = 0
+total_restored_balance = 0
+total_filtered_balance = 0
+
+# Importing the EON accounts
+for account in eon_dump_data['accounts']:
```

Comment:
Cosmetic: consider using:
`for account, account_data in eon_dump_data["accounts"].items():`

Reply:
Done